src/VisualStudio/VisualStudioInteractiveComponents/AssemblyRedirects.cs

```diff
@@ -5,17 +5,17 @@
 
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.Scripting.dll")]
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.CSharp.Scripting.dll")]
-[assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
+// [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
```

Comment:
If we want to preserve F5 for VB then we're going to need to close out this PR.  This component has never built with the proper dependency set.  Originally @mattwar and I tried to fix this by setting the project up to build correctly.  That leads us down another rabbit hole of problems (VSIX bloating for instance).  The scripting team signed off on removing VB as it's not currently supported as a scenario.  Fixing this to build properly isn't something I have time for in RC.